src/gerbil/gxi

```diff
@@ -2,10 +2,14 @@
 set -eu
 
 if [ -z "${GERBIL_HOME:-}" ]; then
-    GERBIL_HOME=$(dirname $(cd ${0%/*} && echo $PWD))
+    GERBIL_HOME="$(dirname "$(cd ${0%/*} && echo $PWD)")"
     export GERBIL_HOME
 fi
 
+if [ -z "${GERBIL_GSI:-}" ]; then
+    source "${GERBIL_HOME}/src/conf.sh"
```

Comment:
This is a little problematic with out of tree installations -- the src directory might not exist.
I think we might want to put it in lib instead, as part of the build script.